server/apps/publish/archive_publish.py

```diff
@@ -318,8 +318,8 @@ def publish(self, doc, updates, target_media_type=None):
 
                 # Step 2(a)(i)
                 subscribers = list(get_resource_service('subscribers').get(req=None, lookup=None))
-                recipients = [s['email'] for s in subscribers]
-                send_article_killed_email(doc, recipients, queued_items[0].get('completed_at'))
+                recipients = [s.get('email') for s in subscribers]
```

Comment:
We could even avoid that by adding the filtering condition to the expression itself, making the use of `filter()` unnecessary, what do you think?

```py
recipients = list(s.get('email') for s in subscribers if s.get('email'))
```